src/sage/numerical/interactive_simplex_method.py

```diff
@@ -2611,9 +2620,17 @@ def run_revised_simplex_method(self):
         self._final_revised_dictionary = d
         return HtmlFragment("\n".join(output))
 
-    def run_simplex_method(self):
+    def run_simplex_method(self, pivot_rule = None, *args):
         r"""
         Apply the simplex method and return all steps and intermediate states.
+        
+        INPUT:
+        
+        - :string: or :SimplexMethodPivot: Optional- set a pivot rule. 
+          Supported rules are "steepest_edge", "blands_rule", "dantzig", "NW_rule".
```

Comment:
Formatting for literal strings:
```suggestion
          Supported rules are ``"steepest_edge"``, ``"blands_rule"``, ``"dantzig"``, ``"NW_rule"``.
```